__tests__/Utilities.test.js

```diff
@@ -110,6 +112,58 @@ describe('Utilities', () => {
 
   })
 
+  describe('generateTitle()', () => {
+
+    it('returns a label', () => {
+      const property = {
+        "propertyLabel": "Instance of",
+        "propertyURI": "http://id.loc.gov/ontologies/bibframe/instanceOf"
+      }
+      const labelProperty = renderer.create(generateTitle(property)).toJSON()
+      expect(labelProperty).toEqual("Instance of")
+    })
+
+    it.only('returns a label and a Required Superscript', () => {
```

Comment:
whoops - bet you didn't mean to leave in this .only!

Reply:
being removed